docker/config/local_dev_submitter.env

```diff
@@ -9,10 +9,11 @@
 # ----------------------------------------------------------------------------------
 
 destination.crashstorage_class=socorro.submitter.breakpad_submitter_utilities.BreakpadPOSTDestination
-destination.url=http://antenna:8888/
```

Comment:
This value was wrong. So this change both changes the configuration key and also sets it to the right value. It's port 8888 if you're connecting from the host, but port 8000 if one container is connecting to another.